Kernel/Arch/aarch64/ASM_wrapper.h

```diff
@@ -16,36 +16,36 @@ namespace Kernel::Aarch64::Asm {
 
 inline void set_ttbr1_el1(FlatPtr ttbr1_el1)
 {
-    asm("msr ttbr1_el1, %[value]" ::[value] "r"(ttbr1_el1));
+    asm volatile("msr ttbr1_el1, %[value]" ::[value] "r"(ttbr1_el1));
 }
 
 inline void set_ttbr0_el1(FlatPtr ttbr0_el1)
 {
-    asm("msr ttbr0_el1, %[value]" ::[value] "r"(ttbr0_el1));
+    asm volatile("msr ttbr0_el1, %[value]" ::[value] "r"(ttbr0_el1));
 }
 
 inline FlatPtr get_ttbr0_el1()
 {
     FlatPtr ttbr0_el1;
-    asm("mrs %[value], ttbr0_el1\n"
-        : [value] "=r"(ttbr0_el1));
+    asm volatile("mrs %[value], ttbr0_el1\n"
+                 : [value] "=r"(ttbr0_el1));
     return ttbr0_el1;
 }
 
 inline void set_sp_el1(FlatPtr sp_el1)
 {
-    asm("msr sp_el1, %[value]" ::[value] "r"(sp_el1));
+    asm volatile("msr sp_el1, %[value]" ::[value] "r"(sp_el1));
 }
 
 inline void set_tpidr_el0(FlatPtr tpidr_el0)
 {
-    asm("msr tpidr_el0, %[value]" ::[value] "r"(tpidr_el0));
+    asm volatile("msr tpidr_el0, %[value]" ::[value] "r"(tpidr_el0));
 }
 
 inline void flush()
 {
-    asm("dsb ish");
-    asm("isb");
+    asm volatile("dsb ish");
```

Comment:
This is a no-op, simple asm statements are implicitly volatile (cf https://gcc.gnu.org/onlinedocs/gcc/Extended-Asm.html#Volatile , https://gcc.gnu.org/onlinedocs/gcc/Extended-Asm.html , https://gcc.gnu.org/onlinedocs/gcc/Basic-Asm.html#Basic-Asm)